schemas/questions/types/calculated.json

```diff
@@ -109,6 +109,28 @@
           ]
         }
       },
+      "validation": {
+        "type": "object",
+        "properties": {
+          "messages": {
+            "TOTAL_SUM_NOT_EQUALS": {
+              "$ref": "https://eq.ons.gov.uk/common_definitions.json#/non_empty_string"
+            },
+            "TOTAL_SUM_NOT_LESS_THAN": {
+              "$ref": "https://eq.ons.gov.uk/common_definitions.json#/non_empty_string"
+            },
+            "TOTAL_SUM_NOT_GREATER_THAN": {
+              "$ref": "https://eq.ons.gov.uk/common_definitions.json#/non_empty_string"
+            },
+            "TOTAL_SUM_NOT_GREATER_THAN_OR_EQUALS": {
+              "$ref": "https://eq.ons.gov.uk/common_definitions.json#/non_empty_string"
+            },
+            "TOTAL_SUM_NOT_LESS_THAN_OR_EQUALS": {
+              "$ref": "https://eq.ons.gov.uk/common_definitions.json#/non_empty_string"
+            }
+          }
```

Comment:
Need to add `"additionalProperties": false`, applies to all the ones that were introduced.

Reply:
Good spot - this actually caught an invalid validation dict in the schema introduced.